### PR TITLE
fix: Lightning Nodes i18, remove link from headline

### PR DESCRIPTION
**Pull request by kilrau**
- node*s* (plural)
- remove link in the headline, with public explorers it can only link to one node and it was broken on testnet

### CODE REVIEW

File: src/pages/Hero.tsx [hunk 1]
```diff
@@ -27,10 +25,6 @@ export const Hero = () => {
     const { hideHero, setHideHero, t, denomination, separator } =
         useGlobalContext();
 
-    const openNodeInfo = async () => {
-        window.open(`${config.lightningExplorerUrl}/${nodePubkey()}`, "_blank");
```
Comment (michael1011) [inline src/pages/Hero.tsx:31]: This was the only place we used `lightningExplorerUrl`. Also remove it from `config.ts`

File: src/pages/Hero.tsx [hunk 2]
```diff
@@ -43,9 +37,6 @@ export const Hero = () => {
                 getNodeStats(BTC),
             ]);
             log.debug("node", nodesRes);
```
Comment (michael1011) [inline src/pages/Hero.tsx:39]: If you remove this log statement, we don't need to fetch `nodeRes` at all

**Review by michael1011 — CHANGES_REQUESTED** [review]

File: src/pages/Hero.tsx (hunk 2 above)
Comment (kilrau) [reply src/pages/Hero.tsx]: Done

File: src/pages/Hero.tsx (hunk 1 above)
Comment (kilrau) [reply src/pages/Hero.tsx:31]: Done

**Review by michael1011 — APPROVED** [review]

**Pull request merged**